clash-cores/src/Clash/Cores/LineCoding8b10b.hs

```diff
@@ -0,0 +1,103 @@
+{-# LANGUAGE CPP #-}
```

Comment:
I'm pretty sure there's no CPP in this file...

Reply:
`CLASH_OPAQUE` requires this, right? Or at least the HLS is bugging me about it.

Reply:
Woops, it completely slipped my mind...